packages/core/src/base/staticClient.ts

```diff
@@ -47,11 +62,12 @@ export async function getObjectFromResponse(
  * Returns the entry from the headers list that matches the passed header. The
  * search is case insensitive and the case of the passed header and the list
  * are preserved. Returns the passed header if no match is found.
- *
- * @param header - target header
- * @param headers - list to search from
- *
- * @returns header from the list if there is a match, the header passed otherwise
+ * 
+ * @param header - Target header
+ * @param headers - List to search from
+ * @returns Header from the list if there is a match, the passed header otherwise
+ * 
+ * @export
```

Comment:
What is @export for? If we are not not using, I suggest to remove it

Reply:
It is just to state that it is an exported method. It's just extra information. I saw that we were using it in some of our other doc strings and decided to use it. I don't have any strong feelings about using it or not using it. If you feel it is not very useful I can remove it.

Reply:
does not sound very useful

Reply:
Removed